server/src/config.rs

```diff
@@ -21,6 +21,8 @@ pub struct PdnsOptions {
     pub soa_content: String,
     pub socket_path: Option<String>,
     pub dns_ttl: u32,
+    pub mx_record: String,
```

Comment:
That should be an `Option<String>` no? why mandate pdns to be the authoritative source for the MX field?